src/archana/Ass13_Indexof.java

```diff
@@ -0,0 +1,69 @@
+package archana;
+
+public class Ass13_Indexof {
+
+	char ch;
+	
+
+	void duplicateChar(String name) {
+		for (int i = 0; i < name.length(); i++) {
+			ch = name.charAt(i);
+			if (name.indexOf(ch) == name.lastIndexOf(ch)) {
```

Comment:
you also need to put one more condition which is below

if(i == name.indexOf(ch)){
    SOP(ch)
}